### DISCUSSION TITLE
How to use a gqlgen extension (gqlopentracing)

### DISCUSSION THREAD

**Discussion by chadxzs** [99designs/gqlgen#2670] [Q&A]
Hey! I was looking at https://github.com/99designs/gqlgen-contrib/tree/master/gqlopentracing but it wasn't clear how to actually use an extension. I'm sure we can muddle around and figure it out but it would be great if there were an example or some reference to extensions in the gqlgen docs. Does anyone know of any prior art to look at or documentation we're missing?

**Comment by davidspek** [comment]
I’d recommend using OpenTelemetry since OpenTracing has been deprecated. This repo has the extension for OpenTelemetry and has examples of how to add and implement it. https://github.com/ravilushqa/otelgqlgen
